.github/workflows/ibis-backends.yml

```diff
@@ -393,18 +337,25 @@ jobs:
       - run: python -m pip install --upgrade pip poetry
 
       - name: install ibis
+        if: ${{ matrix.backend.name != 'postgres' }}
```

Comment:
Seems like there are extras defined for postgres as well https://github.com/ibis-project/ibis/blob/master/pyproject.toml#L109